physionet-django/notification/utility.py

```diff
@@ -475,7 +475,7 @@ def contact_applicant(request, application, comments):
 
 
 def contact_reference(request, application, send=True, wordwrap=True,
-                      subject="", body=""):
+                      subject="", body="", cc=""):
```

Comment:
My bad for doing this in the first place, but subject, body, cc should all really be `=None` rather than `=""` because this indicates they are unset.

Reply:
The logic is the same but I can change it